src/main/java/com/aws/greengrass/clientdevices/auth/iot/Thing.java

```diff
@@ -164,11 +164,8 @@ public String getNamespace() {
 
     @Override
     public DeviceAttribute getDeviceAttribute(String attributeName) {
-        if ("ThingName".equals(attributeName)) {
-            return new WildcardSuffixAttribute(thingName);
-        }
-        // TODO: Return other possible DeviceAttributes
-        return null;
+        // TODO: Support other DeviceAttributes
+        return new WildcardSuffixAttribute(thingName);
```

Comment:
from convo: if we want to still have attributeName passed in we'll need to use it, otherwise would need to be reverted back to just `getDeviceAttributes()` that returns a map